platform-core-module/src/main/java/com/softicar/platform/core/module/user/password/UserPasswordGenerator.java

```diff
@@ -52,12 +54,18 @@ public void resetUserPassword(AGUser user) {
 		}
 
 		try (DbTransaction transaction = new DbTransaction()) {
+			AGCoreModuleInstance.getInstance().getEmailServerOrThrow();
 			String password = new UserPasswordGenerator().generatePassword();
 			new UserPasswordUpdater(user, password).update();
 			AGUser.sendPaswordResetNotification(user, password);
+			IDisplayString firstMessagePart;
+			if (showNewPassword) {
+				firstMessagePart = CoreI18n.THE_PASSWORD_FOR_USER_ARG1_IS_NOW_ARG2.toDisplay(user.getLoginName(), password);
+			} else {
+				firstMessagePart = CoreI18n.THE_PASSWORD_FOR_USER_ARG1_HAS_BEEN_RESET.toDisplay(user.getLoginName());
+			}
 			new DomModalAlertDialog(
-				CoreI18n.THE_PASSWORD_FOR_USER_ARG1_IS_NOW_ARG2
-					.toDisplay(user.getLoginName(), password)
+				firstMessagePart//
 					.concat("\n\n")
 					.concat(CoreI18n.IF_AN_EMAIL_SERVER_IS_CONFIGURED_THE_USER_WILL_RECEIVE_THIS_PASSWORD_VIA_EMAIL)).open();
```

Comment:
we don't need the if-part of this sentence if we throw an exception in line 57....but actually, I don't know what this method has to do with the whole "forgot password" mechanism??? there is only one caller for the `resetUserPassword` method

Reply:
oh right, I used this in an old version and forgot to revert the changes
Reverted everything here